scipy/signal/filter_design.py

```diff
@@ -3653,6 +3653,230 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
+def iirnotch(w0, bw, ab=3):
+    """
+    Design second-order IIR notch digital filter.
+
+    A notch filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    Parameters
+    ----------
+    w0 : float
+        Normalized frequency to remove from a signal. It is a
+        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
+        to half of the sampling frequency.
+    bw : float
+        Normalized filter bandwidth (the diference between up and lower
+        passband edges).
+    ab : float, optional
+        Maximum allowed fluctuation of the logarithm gain (attenuation)
```

Comment:
Sorry for being picky. But it seems to me that the word "fluctuation" doesn't fit here, because the gain curve looks simple with a single peak or notch (no fluctuations as such). 

Can you think of another definition or justify that this is the best?